examples/website-demos/social-messaging/src/components/ChannelSearch/SearchResultItem.tsx

```diff
@@ -12,12 +12,12 @@ import {
 
 import { AvatarGroup, getTimeStamp } from '../ChannelPreview/utils';
 import {
-  SocialAttachmentType,
-  SocialChannelType,
-  SocialCommandType,
-  SocialEventType,
-  SocialMessageType,
-  SocialReactionType,
+  // SocialAttachmentType,
+  // SocialChannelType,
+  // SocialCommandType,
+  // SocialEventType,
+  // SocialMessageType,
+  // SocialReactionType,
```

Comment:
are we not using the generics in this component? if not we can delete

Reply:
I just commented these out in this unrelated/unfinished component to limit console warnings